core/src/main/scala/org/genivi/sota/core/WebService.scala

```diff
@@ -120,9 +120,12 @@ class PackagesResource(resolver: ExternalResolverClient, db : Database)
 
   val route = pathPrefix("packages") {
     get {
-      complete {
-        NoContent
-        //Packages.list
+      parameters('regex.?) { (regex) =>
```

Comment:
Currently it returns a 500 and an error message saying what was wrong with the regex. Eg, for the regex "abc)" it would return:

{"error":"Got error 'unmatched parentheses at offset 3' from regexp"}